src/app/public/modules/card/card.component.spec.ts

```diff
@@ -192,17 +192,17 @@ describe('Card component', () => {
     });
   }));
 
-  it('should set the inline delete to the card style when initially added', () => {
+  it('should set the inline delete to the card style when initially added', async(() => {
```

Comment:
Why'd these need to change to `async`?

Reply:
The test wouldn't finish correctly once I disabled the animations.